ipa-core/src/protocol/ipa_prf/boolean_ops/addition_sequential.rs

```diff
@@ -181,13 +175,13 @@ mod test {
             boolean_array::{BA32, BA64},
             U128Conversions,
         },
-        protocol,
         protocol::{
+            self,
```

Comment:
I do not understand this import.

Reply:
The only thing that changed here is that my IDE or rustfmt combined a separate import of the protocol module by name (`use crate::protocol; use crate::protocol::{foo, bar}`) with imports of specific symbols inside the module (`use crate::protocol::{self, foo, bar}`).

But also, this import is only used as `protocol::RecordId`, which seems unnecessary and inconsistent with how we generally do it, so I'll get rid of it.